packages/alfa-css/src/value/keyword.ts

```diff
@@ -16,60 +16,34 @@ const { equals } = Predicate;
  * @public
  */
 export class Keyword<T extends string = string>
-  extends Value<"keyword", false>
+  extends Ident<"keyword", T>
   implements Resolvable<Keyword<T>, never>
 {
   public static of<T extends string>(value: T): Keyword<T> {
     return new Keyword(value);
   }
 
-  private readonly _value: T;
-
   private constructor(value: T) {
-    super("keyword", false);
-    this._value = value;
-  }
-
-  public get value(): T {
-    return this._value;
-  }
-
-  public resolve(): Keyword<T> {
-    return this;
-  }
-
-  public is(...values: Array<string>): boolean {
-    return values.includes(this._value);
+    super("keyword", value);
   }
 
   public equals(value: unknown): value is this {
-    return value instanceof Keyword && value._value === this._value;
-  }
-
-  public hash(hash: Hash): void {
-    hash.writeString(this._value);
+    return value instanceof Keyword && super.equals(value);
   }
 
   public toJSON(): Keyword.JSON<T> {
     return {
       ...super.toJSON(),
-      value: this._value,
     };
```

Comment:
**Suggestion:** `return super.toJSON()` (no need to spread and rebuild the object).
(which says we can probably just remove the `toJSON` here)